src/Voron/Impl/SliceSmallSet.cs

```diff
@@ -313,19 +181,20 @@ bool TryGetValueFromOverflowUnlikely(out TValue value)
 
         public void Clear()
         {
-            Array.Fill(_keySizes, 0);
+            Array.Fill(_keySizes, default);
+            Array.Fill(_keyHashes, default);
```

Comment:
As far as I remember the cleaning of `keyHashes` is wasted effort because the checks do cause the inline check anyways.

Reply:
What do you mean?

Reply:
Currently we don't use `Clear()` method at all so I'd prefer to leave it as is for consistency with other arrays.